traceevents/src/main/java/com/tomtom/kotlin/traceevents/Tracer.kt

```diff
@@ -294,6 +297,25 @@ class Tracer private constructor(
             return null
         }
 
+        /**
+         * Get the parameter names for the method.
+         * For performance reasons, Make sure this code is executed only once per method call.
```

Comment:
Note that if this event is triggered concurrently from different threads for the first time, it might be possible for the `parameterNames` to be calculated twice and one overwriting the other in the `parameterNamesCache.put` call. I think that will be fine, but that does make the comment not entirely correct. Maybe something to note here in the comment. I can imaging some future changes to the code might forget that this can happen..

Reply:
Corrected.